### DISCUSSION TITLE
Missing External Emoji - What am I doing wrong?

### DISCUSSION THREAD

**Discussion by Sub-lime-time** [recursiveGecko/race_bot#9] [General]
As a sys_admin (not developer) type, I have the race_bot up and running, but can't for the life of me get external emoji's to show up on my server. I've gone through channel permissions and searched for possible answers, but still coming up blank.

What am I missing/doing wrong?

<img width="591" alt="image" src="https://github.com/recursiveGecko/race_bot/assets/18699503/3c526e55-7b8e-4f9b-8ae9-865f52f5a556">

Appreciate any insights.

**Comment by recursiveGecko** [comment]
Hey, whoops, that's an oversight.

I've added missing assets and instructions in the latest commit: https://github.com/recursiveGecko/race_bot/commit/7861c386c323776450cf330425416aedef3f9b43

> Emojis from `assets/discord_emojis` need to be uploaded to your own Discord server.
> To find the emoji ID, you can insert the emoji in Discord message box, then type a backslash directly in front of it.
> After you send the message, the emoji ID should appear in chat.

You will have to set the environment variables:

```
DISCORD_EMOJI_ANNOUNCEMENT=<:YOUR_EMOJI_NAME:YOUR_EMOJI_ID>
DISCORD_EMOJI_QUICK=
DISCORD_EMOJI_SPEEDOMETER=
DISCORD_EMOJI_TIMER=
DISCORD_EMOJI_HARD_TYRE=
DISCORD_EMOJI_MEDIUM_TYRE=
DISCORD_EMOJI_SOFT_TYRE=
DISCORD_EMOJI_TEST_TYRE=
DISCORD_EMOJI_WET_TYRE=
DISCORD_EMOJI_INTERMEDIATE_TYRE=
DISCORD_EMOJI_FLAG_YELLOW=
DISCORD_EMOJI_FLAG_RED=
DISCORD_EMOJI_FLAG_CHEQUERED=
```

Let me know if you manage to set it up or if you need any further help.

**Reply by Sub-lime-time** [reply]
Thank YOU! I had started to go down the path of hacking disccord.ex and designing my own graphics just as you posted this :) I'll pull the latest and try again.

**Comment by Sub-lime-time** [comment]
I just wanted to follow-up with you and let you know that things are now working correctly. 

On a separate note, is there something else I need to do to get *radio messages* working?

**Reply by recursiveGecko** [reply]
Glad to hear that. Unfortunately radios are handled by an entirely different system which isn't publicly available right now. I'm not sure when, if ever, those are going to be available in self-hosted instances.

**Reply by Sub-lime-time** [reply]
No worries at all. I know in Multi-view that it's 'experimental' and didn't think the feed gave transcribed radio messages. Just wanted to make sure I was getting the most out of the bot. 

Now I just need some races that I can watch live instead of replays.

Thanks again!
